lib/slimmer/skin.rb

```diff
@@ -10,7 +11,18 @@ def initialize options = {}
       @asset_host = options[:asset_host]
 
       @logger = options[:logger] || NullLogger.instance
-      @strict = options[:strict] || (%w{development test}.include?(ENV['RACK_ENV']))
+
+      @strict = options[:strict]
+      if %w{development test}.include?(ENV['RACK_ENV'])
+        # A Null instance logger passed in a lot,
+        # not too sure of the implications of unwinding
+        # this all as it depends on all apps that use slimmer.
+        # This will make sure everything is logged correctly
+        # and put slimer renderings more in line with rendering
+        # normal templates
```

Comment:
seeing as none of the apps seem to pass it in, it is overridden in app.rb with null instance logger, so the options are either fix all the apps to get the error or use the small hammer of using the rails.logger if it is there